src/main/java/org/openstreetmap/atlas/geography/index/QuadTree.java

```diff
@@ -33,6 +36,21 @@
     private final Quadtree tree = new Quadtree();
     private Rectangle bound;
 
+    public static <K extends Located> QuadTree<K> forLocated(final Collection<K> locatedCollection)
```

Comment:
If you use only `forEach` how about making it more generic and using `Iterable` instead of `Collection`?